app/src/components/competitions/CompetitionWidgets.tsx

```diff
@@ -212,11 +212,18 @@ function CompetitionDuration(props: CompetitionDurationProps) {
   const { showUTC } = props;
 
   const duration = durationBetween(props.startsAt, props.endsAt);
+  const hasThreeDurationSegments = useMemo(
```

Comment:
Do we need to memoize this? Should be a pretty quick operation + shouldn't be ran too frequently I think

Reply:
removed